src/data/customer/__test__/route.test.js

```diff
@@ -37,3 +38,55 @@ it('Retrieving single grouping location', async () => {
   const response = await removeRouteAsync(CONFIG.routeId, token.accessToken);
   expect('data' in response).toBe(true);
 });
+
+test(`Test for update driver schedule`, async () => {
+  const result = getTokenAsync(
+    CONFIG.temail,
+    CONFIG.tpassword,
+    CONFIG.clientId,
+    CONFIG.clientSecret
+  );
+  const token = await result;
+  const playload = {
+    driver_id: 25148,
+    transaction_group_id: 180,
+    start_time: '10:01',
+    end_time: '13:02',
+    start_at: '2018-03-01',
+  }
+  const scheduleId = CONFIG.scheduleId;
+  const response = await updateRouteLocationAsync(scheduleId, playload, token.accessToken);
+  expect(response.status).toBe(200);
+});
+
+test(`Test for delete driver schedule`, async () => {
+  const result = getTokenAsync(
+    CONFIG.temail,
+    CONFIG.tpassword,
+    CONFIG.clientId,
+    CONFIG.clientSecret
+  );
+  const token = await result;
+  const scheduleId = CONFIG.scheduleId;
+  const response = await deleteDriverScheduleAsync(scheduleId, token.accessToken);
+  expect(response.status).toBe(204);
+});
+
+test(`Test for create driver schedule`, async () => {
+  const result = getTokenAsync(
+    CONFIG.temail,
+    CONFIG.tpassword,
+    CONFIG.clientId,
+    CONFIG.clientSecret
+  );
+  const token = await result;
+  const playload = {
+    driver_id: 25148,
```

Comment:
25148 is not a valid driver ID, how did you manage to get this test case passed? lol